lib/utils.js

```diff
@@ -530,5 +531,9 @@ module.exports = {
   getPatientContact: (db, patientShortcodeId, callback) => {
     getPatient(db, patientShortcodeId, true, callback);
   },
+  isValidBooleanExpression: expr =>
```

Comment:
This name implies it does more than it does - really it's just checking that it's a string that's not empty. It could still be an invalid expression.